tests/components/daikin/test_config_flow.py

```diff
@@ -58,7 +64,7 @@ async def test_abort_if_already_setup(hass, mock_daikin):
     MockConfigEntry(domain="daikin", data={KEY_MAC: MAC}).add_to_hass(hass)
 
     result = await flow.async_step_user({CONF_HOST: HOST})
-    assert result["type"] == data_entry_flow.RESULT_TYPE_ABORT
+    assert result["type"] == RESULT_TYPE_ABORT
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5518609)